packages/core/test/configuration.spec.ts

```diff
@@ -20,31 +20,16 @@ describe('configuration', () => {
   })
 
   describe('endpoint overload', () => {
-    it('should not be available for production env', () => {
```

Comment:
why not keeping this one?

Reply:
Because we can't override endpoints via the user configuration anymore